src/compiler/transformers/module/module.ts

```diff
@@ -1786,7 +1807,15 @@ namespace ts {
         scoped: true,
         text: `
             function __export(m) {
-                for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
+                for (var p in m) b(p);
+                function b(p) {
+                    if (!exports.hasOwnProperty(p)) Object.defineProperty(exports, p, {

```

Comment:
This is not down-level compatible with ES3. You should instead feature-test for `Object.create` (since `Object.defineProperty` exists in some versions of IE that only supported it for DOM elements, but `Object.create` did not exist until full ES5 support landed).